extensions/security-webauthn/runtime/src/main/java/io/quarkus/security/webauthn/WebAuthnRunTimeConfig.java

```diff
@@ -222,4 +222,10 @@ public static class RelyingPartyConfig {
      */
     @ConfigItem(defaultValue = "quarkus-credential")
     public String cookieName;
+
+    /**
+     * The cookie path for the session cookies.
+     */
+    @ConfigItem(defaultValue = "/")
```

Comment:
I wonder if we should apply the same rules we applied for various paths previously: `/` would be absolute from the root i.e. an absolute `/` even if the root path is configured, `` (the empty string) which would be the default, would be relative to the configured root path.

That way you could still set the cookie at the root if it is your inclination but the default would be to apply it relatively to the root path.

Does it make sense?

Reply:
Hi @gsmet I recall looking at it, it would break securing non-application path endpoints

Reply:
But now it is all configurable, so the cookie path can be set to the app http root if needed